.github/workflows/NodeJS.yml

```diff
@@ -112,9 +112,6 @@ jobs:
           - isRelease: false
             node: 19
             target_arch: arm64
-          - isRelease: false
```

Comment:
Is this removed on purpose?

Reply:
It was intentional, but I gave it more thought and figured it wouldn't actually be that useful. Reverted now